deployment.rst

```diff
@@ -112,8 +112,22 @@ you'll need to do:
 A) Check Requirements
 ~~~~~~~~~~~~~~~~~~~~~
 
-Use the ``check:requirements`` command to check if your server meets the
-:ref:`technical requirements for running Symfony applications <symfony-tech-requirements>`.
+There are some :ref:`technical requirements for running Symfony applications <symfony-tech-requirements>`.
+In your development machine, the recommended way to check these requirements is
+to use `Symfony CLI`_. However, in your production server you might prefer to
+not install the Symfony CLI tool. In those cases, install this other package in
+your application:
+
+.. code-block:: terminal
+
+    $ composer require symfony/requirements-checker
+
+Then, make sure that the ``auto-scripts`` option of your ``composer.json`` file
+includes the following entry:
+
+.. code-block:: json
+
+    "vendor/bin/requirements-checker": "php-script",
```

Comment:
This will break Pygments as it's invalid JSON. We should at least wrap it in `{ ... }`, but I think showing some context (e.g. `{ "auto-scripts": { "vendor/bin/...": "...." } }`) is maybe also more helpful for readers

Reply:
You are right! It's much better in the way you propose. Done!